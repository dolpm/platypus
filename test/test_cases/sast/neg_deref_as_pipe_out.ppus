pipe test [x: &int] |> int {
  |> @x;
}

pipe main [] |> unit {
    int x = 4;
    test <| [&x];
    |> ();
}