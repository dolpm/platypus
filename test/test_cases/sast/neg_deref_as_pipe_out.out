Fatal error: exception Failure("Can't provide the deref, which is an owned value, as a pipe return value. Please clone the value before returning it.")
